modules/codegen/src/smithy4s/codegen/internals/Renderer.scala

```diff
@@ -563,33 +558,33 @@ private[internals] class Renderer(compilationUnit: CompilationUnit) { self =>
       ),
       obj(
         opNameRef,
-        ext =
-          line"smithy4s.Endpoint[$traitName,${op.renderAlgParams(opObjectName)}]$errorable"
+        ext = line"smithy4s.Endpoint[$traitName,${op.renderAlgParams(opObjectName)}]"
       )(
-        renderId(op.shapeId),
-        line"val input: $Schema_[${op.input}] = ${op.input.schemaRef}.addHints(smithy4s.internals.InputOutput.Input.widen)",
-        line"val output: $Schema_[${op.output}] = ${op.output.schemaRef}.addHints(smithy4s.internals.InputOutput.Output.widen)",
-        renderStreamingSchemaVal("streamedInput", op.streamedInput),
-        renderStreamingSchemaVal("streamedOutput", op.streamedOutput),
-        renderHintsVal(op.hints),
-        line"def wrap(input: ${op.input}) = ${opNameRef}($input)",
-        renderErrorable(op)
+        line"""def schema: $OperationSchema_[${op.renderAlgParams(
```

Comment:
**question:** TLDR why is this a `def` and not a `val`?

Reply:
could be a val, np